test/Microsoft.AspNetCore.Razor.Design.Test/IntegrationTests/ConfigurationMetadataIntegrationTest.cs

```diff
@@ -18,77 +18,53 @@ public ConfigurationMetadataIntegrationTest(BuildServerTestFixture buildServer)
         [InitializeTestProject("SimpleMvc")]
         public async Task Build_WithMvc_AddsConfigurationMetadata()
         {
+            var razorAssemblyInfo = Path.Combine(IntermediateOutputPath, "SimpleMvc.RazorAssemblyInfo.cs");
             var result = await DotnetMSBuild("Build");
 
             Assert.BuildPassed(result);
 
             Assert.FileExists(result, IntermediateOutputPath, "SimpleMvc.Views.dll");
             Assert.FileExists(result, IntermediateOutputPath, "SimpleMvc.Views.pdb");
 
-            Assert.FileExists(result, IntermediateOutputPath, "SimpleMvc.AssemblyInfo.cs");
+            Assert.FileExists(result, razorAssemblyInfo);
             Assert.FileContainsLine(
                 result,
-                Path.Combine(IntermediateOutputPath, "SimpleMvc.AssemblyInfo.cs"),
+                razorAssemblyInfo,
                 "[assembly: Microsoft.AspNetCore.Razor.Hosting.RazorLanguageVersionAttribute(\"2.1\")]");
             Assert.FileContainsLine(
                 result,
-                Path.Combine(IntermediateOutputPath, "SimpleMvc.AssemblyInfo.cs"),
+                razorAssemblyInfo,
                 "[assembly: Microsoft.AspNetCore.Razor.Hosting.RazorConfigurationNameAttribute(\"MVC-2.1\")]");
             Assert.FileContainsLine(
                 result,
-                Path.Combine(IntermediateOutputPath, "SimpleMvc.AssemblyInfo.cs"),
+                razorAssemblyInfo,
                 "[assembly: Microsoft.AspNetCore.Razor.Hosting.RazorExtensionAssemblyNameAttribute(\"MVC-2.1\", \"Microsoft.AspNetCore.Mvc.Razor.Extensions\")]");
         }
 
-        [Fact]
-        [InitializeTestProject("SimpleMvc")]
-        public async Task Build_WithGenerateRazorAssemblyInfo_False_SuppressesConfigurationMetadata()
```

Comment:
This got moved to BuildIntegrationTest since it's not just the configuration metadata now (it's also the related assembly attribute)